### PR TITLE
Parent class

### PR DESCRIPTION
**Pull request by LlamaBack**
created a parent class for stat tracker and created smaller helper methods for the modules

### CODE REVIEW

File: lib/data_loader.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require 'csv'
+require_relative 'game'
+require_relative 'team'
+require_relative 'game_team'
+require_relative './game_processor'
+require_relative './league_processor'
+require_relative './season_processor'
+require_relative './team_processor'
+
+class DataLoader
```
Comment (mballantyne3) [inline lib/data_loader.rb:10]: I love this idea of a parent class for our data fetching methods that were previously taking up seemingly unnecessary real estate

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -1,38 +1,15 @@
...
-require_relative './season_processor'
-require_relative './team_processor'
-
-class StatTracker
+require_relative 'data_loader'
+
+class StatTracker < DataLoader
   include GameProcessor, LeagueProcessor, SeasonProcessor, TeamProcessor
   attr_reader :locations, :data
-  def initialize(game_path, team_path, game_teams_path)
```
Comment (mballantyne3) [inline lib/stat_tracker.rb:13]: so much cleaner looking without all of this here, and still easily accessible to determine where the information is coming from

**Review by mballantyne3 — COMMENTED** [review]
Love the changes Jake, great work on the parent class!

**Review by torlemann — APPROVED** [review]
I'm glad we were able to go the parent route with data loader, cleans up stat tracker setup nicely

**Pull request merged**